lib/param_parsing/environment_test.go

```diff
@@ -97,3 +101,83 @@ func TestGetParamsFromEnvironment_log_level_from_env(t *testing.T) {
 		t.Errorf("Determined log level is not matching. Got %q, expected %q", params.LogLevel, expected)
 	}
 }
+
+func TestNoColorEnvVar(t *testing.T) {
+	envVarName := "NO_COLOR"
+	_ = os.Setenv(envVarName, "true")
+	goCommandArgs := []string{"run", "../../main.go", "--dry-run", "1.10.5"}
+
+	t.Logf("Testing %q env var", envVarName)
+
+	out, err := exec.Command("go", goCommandArgs...).CombinedOutput()
+	if err != nil {
+		t.Fatalf("Unexpected failure: \"%v\", output: %q", err, string(out))
+	}
+
+	_ = os.Unsetenv(envVarName)
+
+	matched, err := regexp.MatchString(ansiCodesRegex, string(out))
+	if err != nil {
+		t.Fatalf("Unexpected failure: \"%v\", output: %q", err, string(out))
+	}
+
+	if matched {
+		t.Errorf("Expected no ANSI color codes in output, but found some: %q", string(out))
+	} else {
+		t.Log("Success: no ANSI color codes in output")
+	}
+}
+
+func TestForceColorEnvVar(t *testing.T) {
+	envVarName := "FORCE_COLOR"
+	if color.SupportColor() {
```

Comment:
Could this logic be inverted and just return early to avoid putting everyuthing into this scope?

Reply:
Yep, good point. Addressing: c783ac3